### PR TITLE
Fix findCallSite for windows and null cp.stdio elements

### PR DESCRIPTION
**Pull request by efokschaner**
On windows, callsite logging wasn't working for me as none of the filepaths matched the regex.
Also in the kitchensink test cp.stdio looked like `[null, null, null]`. This may be an old node thing (i'm on 0.12.4), so i added a check for that.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -215,10 +218,9 @@ function dump() {
         processes.forEach(function (cp) {
             var fds = [ ];
             console.log('  - PID %s', cp.pid);
-            
             if (cp.stdio && cp.stdio.length) {
                 cp.stdio.forEach(function (s) {
-                    if (s._handle && s._handle.fd) { fds.push(s._handle.fd); }
+                    if (s && s._handle && s._handle.fd) { fds.push(s._handle.fd); }
```
Comment (efokschaner) [inline index.js]: Thinking about this.... could s._handle.fd be zero like in the other place?

**Comment by efokschaner** [comment]
https://github.com/myndzi/wtfnode/pull/5/commits/4b6960a4f49fb434d77be680ba03f7394617cc62 changes all the falsey fd checks for safety / consistency.

**Comment by myndzi** [comment]
Good call on `path.sep`! Thanks for digging into this from the Windows end. Note in the end I went with `== null` (not `=== null`) instead of HOP, but either really works. I'll have a test at the handle stuff in nodes 0.10 through 5 when I get a moment.

**Comment by myndzi** [comment]
One comment immediately: please change the require('path') to be with the other requires; the reason the variables are declared, then the hooking stuff, then the requires, is so that wtfnode can get its claws into everything before anything else may hold references to them. Probably not a problem with the `path` module, but that's why I did it that way and could become a problem later possibly.

**Comment by efokschaner** [comment]
Yeah I was wondering about that require order. I'll fix it. Also FYI !== null would not work because it wouldn't catch undefined. Just refreshed myself on the JS equality tests... They're quite insane.

**Comment by myndzi** [comment]
Yeah, that's what I was clarifying :) I try to avoid === at all costs, and jshint doesn't like it either... but in the specific case of == null, it can be useful. Jshint is also why it's double-wrapped, e.g. `((thing == null))`

**Comment by myndzi** [comment]
Your comments remind me... internal modules now can have path separators >:(

I'll patch that up separately. Thanks for your help on this!

**Pull request merged**